frontend/src/features/properties/components/forms/subforms/PagedBuildingForms.tsx

```diff
@@ -33,6 +34,18 @@ const getPageErrors = (errors: any, nameSpace: any) => {
   );
 };
 
+const deletePagedBuilding = (
+  buildings: IFormBuilding[],
+  currentPage: number,
+  setCurrentPage: Function,
```

Comment:
It looks like `setCurrentPage(currentPage - 1)` requires a number parameter.  I would be better to provide that detail here so that it doesn't accept a generic function.